src/api/helpers/index.ts

```diff
@@ -1,29 +1,37 @@
-import { DEVELOPMENT, EXTENSION_ID } from "statics";
+import {
+  DEVELOPMENT,
+  EXTERNAL_MSG_REQUEST,
+  EXTERNAL_MSG_RESPONSE,
+} from "statics";
+import { Response } from "../types";
 
-interface Response {
-  applicationState: string;
-  publicKey: string;
-  mnemonicPhrase: string;
-  isCorrectPhrase: boolean;
-  confirmedPassword: boolean;
-  transactionStatus: string;
-  error: string;
-}
-
-export const sendMessageAndAwaitResponseExternal = (msg: {}): Promise<Response> =>
-  new Promise((resolve) => {
-    chrome.runtime.sendMessage(EXTENSION_ID, msg, (res: Response) =>
-      resolve(res),
-    );
+export const sendMessageToContentScript = (msg: {}): Promise<Response> => {
```

Comment:
renamed these messages to be more indicative of what they are doing in the flow of the app